src/bindgen/utilities.rs

```diff
@@ -261,24 +261,31 @@ impl SynAttributeHelpers for [syn::Attribute] {
     }
 
     fn get_comment_lines(&self) -> Vec<String> {
-        let mut comment = Vec::new();
-
-        for attr in self {
```

Comment:
Is there any reason to rewrite the for loop instead of just changing `comment.push(text)` for `comment.extend(split_doc_attr(content.value())`?

It's not particularly easier to understand IMHO.

Reply:
Agree with you. I'll revert it back to the for-loop version and use Vec::extend instead. Thanks for suggestions!